app/Helpers/database/game.php

```diff
@@ -345,9 +345,9 @@ function getGamesListByDev(
     if (!empty($orderBy)) {
         if (!Str::contains($orderBy, "Title")) {
             if ($sortBy < 10) {
-                $orderBy .= ", Title";
+                $orderBy .= ", IF(gd.Title LIKE '~%',1,0), gd.Title";
             } else {
-                $orderBy .= ", Title DESC";
+                $orderBy .= ", IF(gd.Title LIKE '~%',0,1), gd.Title DESC";
             }
```

Comment:
Replacing this with just ascending sort for unrelated sorting feels more natural to me 🤔 
`$orderBy .= ", IF(gd.Title LIKE '~%',1,0), gd.Title";`